modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -264,6 +266,11 @@ public List<ServiceRegistration<?>> deploy(
 					objectDefinition, _objectEntryService,
 					_objectRelationshipLocalService),
 				null),
+			_bundleContext.registerService(
+				ObjectRelatedModelsPredicateProvider.class,
```

Comment:
Maybe we should register before the `ObjectRelatedModelsProvider` class to keep the alphabetical order

Reply:
Not, my bad, in the new commit!